test/consistency.jl

```diff
@@ -201,13 +201,12 @@ function consistency(problem :: Symbol; nloops=100, rtol=1.0e-8)
   path = dirname(@__FILE__)
   problem_s = string(problem)
   @printf("Checking problem %-15s%20s\t", problem_s, "")
-  include("$problem_s.jl")
   problem_f = eval(problem)
   nlp_ampl = AmplModel(joinpath(path, "$problem_s.nl"))
   nlp_jump = JuMPNLPModel(problem_f())
   nlp_simple = eval(parse("$(problem)_simple"))()
   nlps = [nlp_ampl; nlp_jump; nlp_simple]
-  @unix_only begin
+  @static if is_unix()
     nlp_cutest = CUTEstModel(uppercase(problem_s))
     push!(nlps, nlp_cutest)
   end
```

Comment:
To eliminate the circular dependencies, we should probably eliminate AmplModel and CUTEstModel from here. It's safe to compare a CUTEstModel with a corresponding JuMPNLPModel in CUTEst.jl and an AmplModel with a corresponding JuMPNLPModel in AmplNLReader since those packages rely on NLPModels.jl. What do you think?